db/migrate/20231108172936_add_valid_to_configuration_scripts.rb

```diff
@@ -0,0 +1,6 @@
+class AddValidToConfigurationScripts < ActiveRecord::Migration[6.1]
+  def change
+    add_column :configuration_scripts, :payload_valid, :boolean
```

Comment:
Should we set a default of true (or in Ruby use default_value_for)?  Reason I'm asking is that ansible playbooks will be seen as "nil" valid, which is falsey in Ruby, so `.payload_valid?` will return false.

Reply:
Done